launch/launch/frontend/expose.py

```diff
@@ -98,7 +98,15 @@ def expose_impl_decorator(exposed):
                     name
                 )
             )
-        parse_methods_map[name] = found_parse_method
+        if exposed_type == 'action':
```

Comment:
Do we need to have this condition on the `exposed_type`? According the docstring, `exposed_type` is only used for clearer error logging.

Reply:
There are two kinds of parsing functions, one for substitutions and one for actions.
The wrapper only apply to actions (because substitutions parsing functions are different).

Reply:
Removed the old comment in https://github.com/ros2/launch/pull/468/commits/716e15d729495d6b5e3211cf585600eebe8cdc28.